google/cloud/storage/bucket.py

```diff
@@ -170,6 +170,17 @@ class LifecycleRuleConditions(dict):
     :param number_of_newer_versions: (Optional) Apply rule action to versioned
                                      items having N newer versions.
 
+    :type days_since_custom_time: int
+    :param days_since_custom_time: (Optional) Apply rule action to items whose number of days
+                                   elapsed since the custom timestamp. This condition is relevant
+                                   only for versioned objects. The value of the field must be a non
+                                   negative integer. If it's zero, the object version will become
+                                   eligible for lifecycle action as soon as it becomes custom.
+
+    :type custom_time_before: :class:`datetime.date`
+    :param custom_time_before: (Optional) Apply rule action to items whose custom time is before this
+                                date. This condition is relevant only for versioned objects.
```

Comment:
Please include similar text to noncurrent_time_before with ISO8601 and an example.